ibazel/profiler/profiler.js

```diff
@@ -0,0 +1,97 @@
+// Copyright 2017 The Bazel Authors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+(function(){
+  function extractUrl() {
+    var url;
+    [...window.document.getElementsByTagName('script')].forEach(function (element) {
+      if (!url && element.src && element.src.match(/^[^:]+:\/\/(.*)\/z?profiler\.js(?:\?(.*))?$/)) {
+        const parsed = document.createElement('a');
+        parsed.href = element.src;
+        url = `${parsed.protocol}//${parsed.host}/profiler-event`;
+      }
+    });
+    return url;
+  };
+
+  const profilerEventUrl = extractUrl();
+  if (!profilerEventUrl) {
+    this.console.error("iBazel profiler disabled because it could not find its own <SCRIPT> tag");
+    return;
+  }
+
+  if (!window.navigator.sendBeacon) {
+    console.error("iBazel profiler disabled because beacon API is not available")
```

Comment:
Can you add a link to the README.md and some words in README about what a beacon is so that if you get this error you have some next steps?

Reply:
👍